include/tables/scene_table.h

```diff
@@ -0,0 +1,130 @@
+ *    - Argument 1: Enum value for this scene
+ *    - Argument 2: Title card's text id, or zero for no title card
+ *    - Argument 3: Scene draw config index
+ *    - Argument 4: Restriction flags. See `RESTRICTIONS_SET` on z_parameter.c
+ *    - Argument 5: Persistent cycle flag: switch0
+ *    - Argument 6: Persistent cycle flag: switch1
+ *    - Argument 7: Persistent cycle flag: chest
+ *    - Argument 8: Persistent cycle flag: collectible
+ *
+ * DEFINE_ACTOR_UNSET is needed to define empty entries from the original game
+ *    - Argument 0: Enum value for this scene
+ */
+/* 0x00 */ DEFINE_SCENE(Z2_20SICHITAI2, SCENE_20SICHITAI2, 0x0116, SCENE_DRAW_CFG_MAT_ANIM, RESTRICTIONS_NONE, (1 << 10) | (1 << 11), 0, 0, (1 << 30)) // Southern Swamp (Clear)
+/* 0x01 */ DEFINE_SCENE_UNSET(SCENE_UNSET_01)
+/* 0x02 */ DEFINE_SCENE_UNSET(SCENE_UNSET_02)
+/* 0x03 */ DEFINE_SCENE_UNSET(SCENE_UNSET_03)
+/* 0x04 */ DEFINE_SCENE_UNSET(SCENE_UNSET_04)
+/* 0x05 */ DEFINE_SCENE_UNSET(SCENE_UNSET_05)
+/* 0x06 */ DEFINE_SCENE_UNSET(SCENE_UNSET_06)
+/* 0x07 */ DEFINE_SCENE(KAKUSIANA, SCENE_KAKUSIANA, 0x0000, SCENE_DRAW_CFG_MAT_ANIM, RESTRICTIONS_NONE, 0, 0, 0, (1 << 2) | (1 << 10) | (1 << 11)) // Lone Peak Shrine & Grottos
+/* 0x08 */ DEFINE_SCENE(SPOT00, SCENE_SPOT00, 0x0000, SCENE_DRAW_CFG_NOTHING, RESTRICTIONS_NONE, 0, 0, 0, 0) // Cutscene Scene
+/* 0x09 */ DEFINE_SCENE_UNSET(SCENE_UNSET_09)
+/* 0x0A */ DEFINE_SCENE(Z2_WITCH_SHOP, SCENE_WITCH_SHOP, 0x011A, SCENE_DRAW_CFG_MAT_ANIM, RESTRICTIONS_INDOORS, 0, 0, 0, 0) // Magic Hags' Potion Shop
+/* 0x0B */ DEFINE_SCENE(Z2_LAST_BS, SCENE_LAST_BS, 0x0000, SCENE_DRAW_CFG_MAT_ANIM, RESTRICTIONS_MOON, 0, 0, 0, (1 << 31)) // Majora's Lair
+/* 0x0C */ DEFINE_SCENE(Z2_HAKASHITA, SCENE_HAKASHITA, 0x0113, SCENE_DRAW_CFG_MAT_ANIM, RESTRICTIONS_NONE, 0, 0, 0, (1 << 31)) // Beneath the Graveyard
+/* 0x0D */ DEFINE_SCENE(Z2_AYASHIISHOP, SCENE_AYASHIISHOP, 0x010E, SCENE_DRAW_CFG_MAT_ANIM, RESTRICTIONS_INDOORS, 0, 0, 0, 0) // Curiosity Shop
+/* 0x0E */ DEFINE_SCENE_UNSET(SCENE_UNSET_0E)
+/* 0x0F */ DEFINE_SCENE_UNSET(SCENE_UNSET_0F)
+/* 0x10 */ DEFINE_SCENE(Z2_OMOYA, SCENE_OMOYA, 0x0132, SCENE_DRAW_CFG_DEFAULT, RESTRICTIONS_SET(0, 0, 0, 0, 0, 3, 0, 3, 0, 0, 0, 0), 0, 0, 0, 0) // Mama's House (Ranch House in PAL) & Barn
+/* 0x11 */ DEFINE_SCENE(Z2_BOWLING, SCENE_BOWLING, 0x0108, SCENE_DRAW_CFG_MAT_ANIM, RESTRICTIONS_INDOORS, 0, 0, 0, 0) // Honey & Darling's Shop
+/* 0x12 */ DEFINE_SCENE(Z2_SONCHONOIE, SCENE_SONCHONOIE, 0x010B, SCENE_DRAW_CFG_DEFAULT, RESTRICTIONS_INDOORS, 0, 0, 0, 0) // The Mayor's Residence
+/* 0x13 */ DEFINE_SCENE(Z2_IKANA, SCENE_IKANA, 0x0141, SCENE_DRAW_CFG_MAT_ANIM, RESTRICTIONS_NONE, (1 << 20), 0, 0, (1 << 1) | (1 << 30)) // Ikana Canyon
+/* 0x14 */ DEFINE_SCENE(Z2_KAIZOKU, SCENE_KAIZOKU, 0x0000, SCENE_DRAW_CFG_MAT_ANIM, RESTRICTIONS_NONE, 0, 0, 0, 0) // Pirates' Fortress
+/* 0x15 */ DEFINE_SCENE(Z2_MILK_BAR, SCENE_MILK_BAR, 0x010C, SCENE_DRAW_CFG_MAT_ANIM, RESTRICTIONS_INDOORS, 0, 0, 0, 0) // Milk Bar
```

Comment:
```suggestion
/* 0x15 */ DEFINE_SCENE(Z2_MILK_BAR, SCENE_MILK_BAR, 0x10C, SCENE_DRAW_CFG_MAT_ANIM, RESTRICTIONS_INDOORS, 0, 0, 0, 0) // Milk Bar
```
Do we need these leading 0's? Could `0x0000` just be `0` as well?